### PR TITLE
Add (optional) logger to log failed token fetch requests

### PR DESCRIPTION
**Pull request by frankverhoeven**
Allows a logger to be injected so failed token requests can be logged.
Also updated phpcs to use the mos/cs package.

### CODE REVIEW

File: src/TokenManager/Jwt.php [hunk 1]
```diff
@@ -21,6 +23,9 @@ final class Jwt implements TokenManagerInterface
      */
     private $jwtParser;
 
+    /** @var LoggerInterface */
```
Comment (Feijs) [inline src/TokenManager/Jwt.php]: New docblock format?

**Review by Feijs — COMMENTED** [review]
Where are the logs written to in this case?

File: src/TokenManager/Jwt.php (hunk 1 above)
Comment (frankverhoeven) [reply src/TokenManager/Jwt.php]: Nope, phpstorm not using the right config

**Comment by frankverhoeven** [comment]
> Where are the logs written to in this case?

Nowhere, this has to configured in the app using this package (the monolith)

**Comment by Feijs** [comment]
> Nowhere, this has to configured in the app using this package (the monolith)

:friday:

**Review by Feijs — APPROVED** [review]

**Pull request merged**